### DISCUSSION TITLE
Assigner une ville à une guilde

### DISCUSSION THREAD

**Discussion by Chevalier2** [Crownicles/Crownicles#590] [Nouvelles idées]
Pouvoir assigner une ville à notre guide dans un but purement RP.

C'est à dire acheter un emplacement de guilde dans une ville dans le guild shop. Et avoir pourquoi pas une interaction avec le nom de la guilde lorsque l'on traverse une ville.

**Comment by niqore** [comment]
Fortement lié à #577

**Reply by Pietagorh** [reply]
Ouaip
